samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', filepath = 'path/to/image.png') {
+  // [START dlp_inspect_image_file]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const filepath = 'path/to/image.png';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const DLP = require('@google-cloud/dlp');
+
+  // Import other required libraries
+  const fs = require('fs');
+
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  async function inspectImageFile() {
+    // Get the bytes of the file
+    const fileBytes = Buffer.from(fs.readFileSync(filepath)).toString('base64');
```

Comment:
Users copy/paste code :/ Can you please use `promisify` instead of doing a sync file read?

Reply:
Done